x-pack/plugins/apm/public/components/app/mobile/service_overview/geo_map/embedded_map.tsx

```diff
@@ -23,7 +23,7 @@ import { css } from '@emotion/react';
 import { i18n } from '@kbn/i18n';
 import { EuiText } from '@elastic/eui';
 import type { Filter } from '@kbn/es-query';
-import { useDataViewId } from '../../../../../hooks/use_data_view_id';
+import { useStaticDataViewId } from '../../../../../hooks/use_static_data_view_id';
```

Comment:
Is the plan to use ad hoc data view for this also (and everywhere else we call `useStaticDataViewId `?)

Reply:
I would love to use the ad hoc data views for all the integrations with the embeddable (dashboards, lens etc). 

Now we use the static data views 
1. Mobile map
2. Discover links
3. Diagnostic page

I intended to narrow down the focus of this pull request specifically to resolve the metrics tab issues. However, it might be valuable to update them all at once. I'll give it a shot, and hopefully, there won't be any surprises.